oat-sa/extension-tao-itemqti
# Feat/aut 2190/toggle allow elimination

related to https://oat-sa.atlassian.net/browse/AUT-2190

### Description 

PR makes allowElimination property configurable through `config/tao/client_lib_config_registry.conf.php`

### How to test 

add `'taoQtiItem/creator/interaction/choice/property/allowElimination' => 'hide',` to `services/features` `visibility section` check config prop no longer exist in choiceInteraction question form
